src/js/playlist/playlist.js

```diff
@@ -47,6 +47,15 @@ export function normalizePlaylistItem(model, item, feedData) {
     return formatItem(playlistItem);
 }
 
+export function wrapPlaylistIndex(index, length) {
```

Comment:
can we write a test for this?

Reply:
https://github.com/jwplayer/jwplayer/pull/3668/commits/b22a2754b515bac16fcc76278685a2c387d84852